modules/nw-sriov-dual-nic-con.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: CONCEPT
+[id="nw-sriov-dual-nic-con_{context}"]
+= NIC partitioning for SR-IOV devices (Technology Preview)
+
+{product-title} can be deployed on a server with a dual port network interface card (NIC).
+You can partition a single, high-speed dual port NIC into multiple virtual functions (VFs) and enable SR-IOV.
+The following diagram shows how this works:
+
+
+When you partition a single, high-speed NIC into multiple VFs, you can use the NIC for both control and data plane traffic.
+
+This feature supports the use of bonds for high availability with the Link Aggregation Control Protocol (LACP).
+
+[NOTE]
+====
+Only one LACP can be declared by physical NIC.
+====
+
+Cloud-native network functions are able to use SR-IOV virtual functions. This topology supports OVN-Kubernetes on SR-IOV virtual functions (VFs).
```

Comment:
This paragraph is disconnected from the rest of the documentation. Consider either rephrasing or removing it.